modules/swagger-codegen-maven-plugin/README.md

```diff
@@ -45,7 +45,8 @@ mvn clean compile
 - `modelPackage` - the package to use for generated model objects/classes
 - `apiPackage` - the package to use for generated api objects/classes
 - `invokerPackage` - the package to use for the generated invoker objects
-- `modelNamePrefix` and `modelNameSuffix` - Sets the pre- or suffix for model classes and enums.
+- `modelNamePrefix` and `modelNameSuffix` - Sets the pre- or suffix for model classes and enums
+- `useJaxbAnnotations` - enable Jaxb annotations inside the generated models
```

Comment:
@danmikita I found 2 other options starting with `use` and no option starts with `enable` at the moment so I'm ok with the name you pick. 

I'll perform a bit more tests later today and merge into master if no question/feedback from anyone.